### PR TITLE
Autoset CRAN mirror and working directory when /app is present

### PR DESCRIPTION
**Pull request by rpkyle**
Previously, users had to invoke `setwd` within their `app.R` to properly set the working directory for deployed Dash for R apps. The proposed modifications do this automatically, if the `/app` folder exists  at the root level of the deployed app's folder structure.

### CODE REVIEW

File: Dockerfile-18 [hunk 1]
```diff
@@ -23,8 +23,7 @@ RUN apt-get -qy install \
       autoconf \
       libtool \
       debootstrap \
-      systemd \
-      ccache
```
Comment (josegonzalez) [inline Dockerfile-18:27]: Why did you remove `ccache`?

Comment (rpkyle) [reply Dockerfile-18:27]: Because I added it in my efforts to speed up package build performance. I was not able to discern that it had any demonstrable impact on compile times during the build step.

File: Dockerfile-18 [hunk 2]
```diff
@@ -120,9 +120,13 @@ RUN fakechroot fakeroot chroot $CHROOT_DIR \
...
+  /usr/bin/R -e "install.packages(c('devtools', 'callr', 'data.table', 'plotly', 'ggplot2', 'scales', 'httr', 'jsonlite',  'magrittr', 'digest', 'viridisLite', 'base64enc', 'htmltools', 'htmlwidgets', 'tidyr', 'hexbin', 'RColorBrewer', 'dplyr', 'tibble', 'lazyeval', 'rlang', 'crosstalk', 'purrr', 'promises', 'R6', 'shiny', 'assertthat', 'glue', 'pkgconfig', 'Rcpp', 'tidyselect', 'BH', 'plogr', 'gtable', 'MASS', 'mgcv', 'reshape2', 'withr', 'lattice', 'yaml', 'curl', 'mime', 'openssl', 'later', 'labeling', 'munsell', 'cli', 'crayon', 'fansi', 'pillar', 'ellipsis', 'stringi', 'vctrs', 'lifecycle', 'nlme', 'Matrix', 'colorspace', 'askpass', 'utf8', 'plyr', 'stringr', 'httpuv', 'xtable', 'sourcetools', 'backports', 'zeallot', 'sys', 'fiery', 'uuid', 'future', 'reqres', 'globals', 'listenv', 'urltools', 'brotli', 'xml2', 'webutils', 'codetools', 'triebeard')); remotes::install_github('plotly/dashR', upgrade=TRUE)"
```
Comment (josegonzalez) [inline Dockerfile-18]: Can you explain this change as well?

Comment (rpkyle) [reply Dockerfile-18]: I'll reconfirm the package list later this evening, but the mirror setting was moved to an earlier line in `Dockerfile-18`.

https://github.com/plotly/heroku-buildpack-r-build/blob/65ec54d8065acd955c22652a91b3cbde0a758e21/Dockerfile-18#L123-L125

Comment (rpkyle) [reply Dockerfile-18]: @josegonzalez Sorry for the delay, but I confirm that the package list is the same. We've added `lubridate`, and specified that we should deploy the release version from `master` in lieu of deploying with `dev`.

**Review by josegonzalez — APPROVED** [review]
